packages/atomic/src/pages/examples/insights.html

```diff
@@ -123,13 +127,17 @@
                     tooltip="post to feed"
                     action="postToFeed"
                   ></atomic-insight-result-action>
+                  <atomic-insight-result-attach-to-case></atomic-insight-result-attach-to-case>
                   <atomic-insight-result-action tooltip="quickview" action="quickview"></atomic-insight-result-action>
                 </atomic-insight-result-action-bar>
                 <atomic-result-section-badges>
                   <atomic-field-condition must-match-sourcetype="Salesforce">
                     <atomic-result-badge label="Salesforce" class="salesforce-badge"></atomic-result-badge>
                   </atomic-field-condition>
                 </atomic-result-section-badges>
+                <atomic-result-section-actions
+                  ><atomic-insight-result-attach-to-case read-only="true"></atomic-insight-result-attach-to-case
```

Comment:
I had this UI issue when there was no badge. Not that big of a problem imo

https://github.com/user-attachments/assets/1676bb9b-f7f8-4d39-b15a-2cc2429b7551

Reply:
Looks like a pretty big issue 😆